docs/user/running-cincinnati.md

```diff
@@ -0,0 +1,71 @@
+# Running Cincinnati
+
+## Prerequisite
+
+APP-SRE publishes the Cincinnati container images at [Quay registry](https://quay.io/repository/app-sre/cincinnati)
+
+We need an image tag from the [Quay registry](https://quay.io/repository/app-sre/cincinnati) to use with the deployment config.
+
+The Cincinnati deploymentconfig present in the git repository at [dist/openshift/](https://github.com/openshift/cincinnati/blob/master/dist/openshift/cincinnati.yaml)
+
+### Steps
+
+* Create a new namespace for Cincinnati
+
+```shell
+oc new-project cincinnati
+```
+
+* Create Cincinnati deployment
+
+```shell
+export IMAGE_TAG=<TAG>
+oc new-app -f cincinnati.yaml \
+  -p IMAGE_TAG=${IMAGE_TAG} \
+```
+
+* Wait for teh deployment config to rollout
+
+```shell
+oc wait --for=condition=available --timeout=5m deploymentconfig/cincinnati
+
+```
+
+* Create an route to the Cincinnati policy engine
+
+Example of a yaml for an ingress route to the policy engine
+
+```json
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: cincinnati
+  namespace: cincinnati
+  labels:
+    app: cincinnati-policy-engine
+  annotations:
+    openshift.io/host.generated: 'true'
+spec:
+  host: cincinnati-cincinnati.apps.ci-ln-13sjybk-d5d6b.origin-ci-int-aws.dev.rhcloud.com
+  to:
+    kind: Service
+    name: cincinnati-policy-engine
+    weight: 100
+  port:
+    targetPort: policy-engine
+  tls:
+    termination: edge
+  wildcardPolicy: None
+status:
+  ingress:
+  - conditions:
+    host: cincinnati-cincinnati.apps.ci-ln-13sjybk-d5d6b.origin-ci-int-aws.dev.rhcloud.com
+    routerCanonicalHostname: apps.ci-ln-13sjybk-d5d6b.origin-ci-int-aws.dev.rhcloud.com
+    routerName: default
+    wildcardPolicy: None
+```
+
+To get existing ingress routes, run:
+```shell
+oc get svc -n openshift-ingress
+```
```

Comment:
Let's add a subsection for changing registry/repo with an example like the one you did [here](https://github.com/LalatenduMohanty/cincinnati/blob/ota-97-deployment-config-changes/hack/deploy_cincinnati.sh). I tested it with:
...
`-p GB_PLUGIN_SETTINGS="$(cat <<-EOF
      [[plugin_settings]]
      name = "release-scrape-dockerv2"
      registry = "registry-1.docker.io"
      repository = "jottofar/openshift-release-dev_ocp-release"
...`
I was also hoping to be able to use a parameters file as the new-app command as given is quite unwieldy. It would be 'oc new-app -f dist/openshift/cincinnati.yaml --param-file=<FILE>' however I have not been able to get the complex GB_PLUGIN_SETTINGS into the file. It may only be able to handle simple 'key=value' lines.

Reply:
@steveeJ Does Cincinnati supports a parameter file? e.g. `'oc new-app -f dist/openshift/cincinnati.yaml --param-file=<>` ?

Reply:
I believe the GB_PLUGIN_SETTINGS would have to be re-factored/simplified. It appears to be an OpenShift limitation regarding the params file but could also be that I just did not discover the secret sauce to making it work.

Reply:
@LalatenduMohanty I wouldn't know why using `param-file` wouldn't be possible, and AFAIU the template doesn't have an opinion on how the parameters are passed to `oc`.

Reply:
It seems only able to handle simple key=value lines like:
GB_PAUSE_SECS=120

The GB_PLUGIN_SETTINGS requires the more complex line like:
GB_PLUGIN_SETTINGS="[[plugin_settings]] name=release-scrape-dockerv2 registry=quay.io repository=openshift-release-dev/ocp-release fetch_concurrency=16 [[plugin_settings]] name=github-secondary-metadata-scrape github_org=openshift github_repo=cincinnati-graph-data reference_branch=master output_directory=/tmp/cincinnati/graph-data [[plugin_settings]] name=openshift-secondary-metadata-parse data_directory=/tmp/cincinnati/graph-data [[plugin_settings]] name=edge-add-remove"
ENVIRONMENT_SECRETS="{}"

I tried lots of different iterations on the above and diidn't come up with a format that was accepted by new-app.